targets/Targets.csproj

```diff
@@ -12,7 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Bullseye" Version="2.2.0" />
-    <PackageReference Include="SimpleExec" Version="4.0.0-beta.1" />
+    <PackageReference Include="SimpleExec" Version="3.0.0" />

```

Comment:
Why was this bumped to 4.0.0-beta.1 before and now back to 3.0.0?

Reply:
Weird. I don't know why this happened! I'll revert.